skrifa/Cargo.toml

```diff
@@ -8,9 +8,15 @@ repository = "https://github.com/googlefonts/fontations"
 readme = "README.md"
 categories = ["text-processing", "parsing", "graphics"]
 
```

Comment:
I think it’d be better to remove the feature and only add serde to the dev-dependency section. This will make it available for tests but not expose it to the public API.

Reply:
Thanks for the help with that, yes, that's better. Had some trouble figuring out the right syntax and what would be the best way to do that here. Updated, should be done.